binding/java/src/ai/picovoice/rhino/Utils.java

```diff
@@ -158,6 +158,10 @@ private static String getArchitecture() throws RuntimeException {
                 return "x86_64";
             }
         } else if (arch.equals("arm") || arch.equals("aarch64")) {
+            if (ENVIRONMENT_NAME.equals("mac")) {
```

Comment:
Can you make the same env <-> arch logic switch here as well?

Reply:
Yup!